### ISSUE TITLE
support for Picooc Mini scale

### ISSUE THREAD

**Issue by artsight** [oliexdev/openScale#401]

**Comment by oliexdev** [comment]
Please provide more information regarding to https://github.com/oliexdev/openScale/wiki/How-to-reverse-engineer-a-Bluetooth-4.x-scale

**Comment by artsight** [comment]
> Please provide more information regarding to https://github.com/oliexdev/openScale/wiki/How-to-reverse-engineer-a-Bluetooth-4.x-scale

Added both files, sorry, was not able to do that from mobile.

**Comment by oliexdev** [comment]
the true values are missing for both files

**Comment by artsight** [comment]
> the true values

sorry, what you mean by that? I see the logfiles are not empty, what is missing?

unfortunately I don't have wireshark to look into bt snoop deeply

**Comment by artsight** [comment]
Here is screenshot from native app with all values. Tried to find them in the btsnoop log but ...
![1550507909643](https://user-images.githubusercontent.com/47692632/52965382-b88f3180-33b5-11e9-8852-61c1d759246f.jpg)

**Comment by oliexdev** [comment]
is this the screenshot from the above log file? Please do it again with different weights described in https://github.com/oliexdev/openScale/wiki/How-to-reverse-engineer-a-Bluetooth-4.x-scale (see step 1.6)

**Comment by artsight** [comment]
yes, the screenshot above is based on the logfile attached previously.

I made 4 other measurements with different weights, please see the attached zip.
[picooc mini logs.zip](https://github.com/oliexdev/openScale/files/2879331/picooc.mini.logs.zip)

**Comment by oliexdev** [comment]
which screenshots belongs to which log?

**Comment by artsight** [comment]
> which screenshots belongs to which log?
Sorry, didn't mention that:
2,3,4.jpg - btsnoop_hci.20190219092631
5.jpg - btsnoop_hci.20190219092904

**Comment by 2m** [comment]
I am also interested in support for this scale. Is there another BT capture required for this to be completed?

**Comment by plant7403** [comment]
Anyone? 😁 please?
As it seems almost all of the models of picooc have very similar ways they synchronize
So it's possible to add like a lot of scales at once with this one

**Comment by oliexdev** [comment]
Since there has been no progress on this issue for a long time, I am closing it for now.  

Please note that implementation of new scale support requires access to the **actual scale**.

If someone wants to continue working on this, feel free to **reopen this issue** or submit a working implementation via a **pull request**.

For guidance on how to add a new scale handler into openScale, see the updated wiki page:  **[How to support a new scale](https://github.com/oliexdev/openScale/wiki/How-to-support-a-new-scale)**
